justspacesapp/static/css/custom.css

```diff
@@ -0,0 +1,268 @@
+:root {
+  --sidebar-background: #fafafa;
+  --sidebar-text: black;
+  --sidebar-text-hover:  black;
+}
+
+body {
+    background: #fafafa;
+    padding-top: 0px;
+}
+
+p {
+    font-size: 1.1em;
+    font-weight: 300;
+    line-height: 1.7em;
+    color: #999;
+}
+
+a,
+a:hover,
+a:focus {
+    color: inherit;
+    text-decoration: none;
+    transition: all 0.3s;
+}
+
+.navbar {
+    padding: 15px 10px;
+    background: #fff;
+    border: none;
+    border-radius: 0;
+    margin-bottom: 40px;
+    box-shadow: 1px 1px 3px rgba(0, 0, 0, 0.1);
+}
+
+.navbar-btn {
+    box-shadow: none;
+    outline: none !important;
+    border: none;
+}
+
+.line {
+    width: 100%;
+    height: 1px;
+    border-bottom: 1px dashed #ddd;
+    margin: 40px 0;
+}
+
+.list-unstyled {
+  padding-left: 30px;
+}
+
+i,
+span {
+    display: inline-block;
+}
+
+/* ---------------------------------------------------
+    SIDEBAR STYLE
+----------------------------------------------------- */
+
+.wrapper {
+    display: flex;
+    align-items: stretch;
+}
+
+#sidebar {
+    min-width: 250px;
+    max-width: 250px;
+    background: var(--sidebar-background);
+    color: var(--sidebar-text);
+    transition: all 0.3s;
+    border-right: 1px solid black
+}
+
+#sidebar.active {
+    min-width: 80px;
+    max-width: 80px;
+    text-align: center;
+}
+
+#sidebar.active .sidebar-header h3,
+#sidebar.active .CTAs {
+    display: none;
```

Comment:
Would it be best to use the more modular Bootstrap 4 utilities for these styles (`display: none`, `display:block`, `text-align`)? It's certainly more conventional to follow the Bootstrap paradigm, but I am not sure how many individual elements you'd have to handle.

https://getbootstrap.com/docs/4.0/utilities/display/#notation

Reply:
These styles show and hide the "Just Spaces" name and logo based on whether or not the menu is collapsed (ie, has the `active` class). I'm not sure how to get at that through bootstrap's styles. Maybe we could pair on that?

Reply:
(Excuse my delayed response!) I think it's fine as it, in that case. No need to fuss with this.